File: 03_20_interesting_polygon/submissions/ted.py
def polygon_area(n):
    new_n = n**2
    old_n_squared = (n-1)**2
    return new_n + old_n_squared


assert polygon_area(1) == 1
assert polygon_area(2) == 5
assert polygon_area(3) == 13
assert polygon_area(4) == 25
print('All Tests Passed')

from tests import full_test
full_test(polygon_area)

File: 06_interesting_polygon/submissions/ted.js
let polygon_area = (n) => {
    var new_n = n ** 2
    var old_n_squared = (n - 1) ** 2
    return (new_n + old_n_squared)
}


let assert = require('assert')
assert.strictEqual(polygon_area(1), 1)
assert.strictEqual(polygon_area(2), 5)
assert.strictEqual(polygon_area(3), 13)
assert.strictEqual(polygon_area(4), 25)
console.log('All Tests Passed')

// Uncomment for complete testing after passing initial tests
let full_test = require('./tests/tests')
full_test(polygon_area)